storybook/index.html

```diff
@@ -338,6 +338,20 @@ <h1 class="sectionHeading">BAMP</h1>
                         </div>
                     </center>
                 </section>
+                <section class="demo" data-module-name="pressModule">
+                    <div class="box box-red">
+                        <div class="pressModule">
+                            <img class="pressModule--img" src="https://i.ytimg.com/vi/f_oKFLeZi88/maxresdefault.jpg" alt="">
+                            <div class="pressModule--header">D4C Funny Valentine
```

Comment:
small thing, re: `pressModule--header` -- technically this is a heading, rather than a header.

"header" is more like a letterhead -- something that appears at or near the top of every page, whereas headings go above paragraphs or sections.

but also since these are news articles, I think it might be better to call it a **headline**? lol